packages/rest/src/router/index.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export {
+  RouteEntry,
```

Comment:
Considering how many items we are exporting, I think it may be best to simply export everything.

```ts
export * from `./routing-table';
```

Thoughts?